beets/importer.py

```diff
@@ -1033,10 +1048,11 @@ def _create(self, task):
         task. If `task` is None, do nothing.
         """
         if task:
-            task.emit_created(self.session)
-            if not task.skip:
-                self.imported += 1
-            return task
+            tasks = task.emit_created(self.session)
+            for task in tasks:
+                if not task.skip:
+                    self.imported += 1
```

Comment:
One alternative here would be to just change the `filefilter` to use the new functionality instead of using the `skip` flag. (That is, it would somehow drop the tasks via returning an empty list instead of mutating the existing task.) Then we wouldn't need to check the `skip` flag here or in the logging stage.